buddyboss/buddyboss-platform
# Shortcode [bbp-topic-form] is not working properly when use in Elementor popup template

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #623 

### How to test the changes in this Pull Request:

1. Make sure to have Elementor and pro version
2. Create a popup, go to Templates > Popups
3. Insert [bbp-topic-form] shortcode then save
4. Go to the frontend where you have made the popup and you will notice that it will not display properly and cannot post anything.

### Proof Screenshots or Video
Fixed editor loading issue and made it work.
http://prntscr.com/tit94e



### Other information:




### Theme PR: 
https://github.com/buddyboss/buddyboss-theme/pull/859
